### DISCUSSION TITLE
How to Initiate the Kevin Assistant or Any Integrated LLM in SysON (SysML v2)?

### DISCUSSION THREAD

**Discussion by pmachadoblatt** [eclipse-syson/syson#774] [Q&A] [answered]
Hi everyone,

Sorry if it looks dumb, but I’m working with the SysON environment and came across mentions of a virtual assistant called "Kevin" or potentially other integrated LLM (Large Language Model) features within the system. However, I haven't been able to find documentation or clear steps on how to initiate or interact with this assistant.
![imagem_2024-10-08_161937920](https://github.com/user-attachments/assets/f0bed9a1-ffb0-43c7-a24f-85d1207471b1)

Could anyone provide guidance on how to start or access "Kevin" or any LLM functionality within SysON? I’ve already checked through some of the services, but I may have missed something. Any pointers or documentation links would be highly appreciated!

Thanks in advance!

**Comment by cbrun** [comment, marked as answer]
Hi there,

Great question, and no worries at all! The "Kevin" assistant you mentioned was actually presented as part of a demo at the Models 2024 conference in Austria. It showcased what can be achieved by extending the SysON environment alongside Sirius Web. Kevin is a small bot designed to provide editing assistance, but it’s important to note that it’s not an official feature of SysON itself—it’s an internal bot we've developed at Obeo for demonstration purposes.

That said, SysON’s architecture does support custom extensions, including possibilities for similar assistants. If developing such functionality is something you're exploring, it's also the kind of tailored service we provide at Obeo, assisting with custom extensions or specific features.

Feel free to reach out if you'd like to discuss this further.

**Reply by pmachadoblatt** [reply]
Hi! I'm a student of Professor Christopher at ITA, in the CONCEPTIO lab, and we're just beginning to transition to SysON. We previously worked extensively with Capella, following the ARCADIA methodology, so I'm still getting familiar with SysML. I found the integration with the assistant very interesting. If you have any study materials or suggestions, or if there's a channel to get in touch, I would really appreciate it!
